net-p2p/p2pool-monero-bin/p2pool-monero-bin-1.8.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+#To prevent confusion with the bitcoin version of p2pool, this ebuild has been named p2pool-monero
+DESCRIPTION="Decentralized pool for Monero mining (Static Binary)"
+HOMEPAGE="https://p2pool.io https://github.com/SChernykh/p2pool"
+SRC_URI="
+	x86?     ( https://github.com/SChernykh/p2pool/releases/download/v${PV}/p2pool-v${PV}-linux-x64.tar.gz )
+	amd64?   ( https://github.com/SChernykh/p2pool/releases/download/v${PV}/p2pool-v${PV}-linux-x64.tar.gz )
+	aarch64? ( https://github.com/SChernykh/p2pool/releases/download/v${PV}/p2pool-v${PV}-linux-aarch64.tar.gz )
+"
```

The AMD64 binary won't run on x86 :(

```suggestion
SRC_URI="
	amd64?   ( https://github.com/SChernykh/p2pool/releases/download/v${PV}/p2pool-v${PV}-linux-x64.tar.gz )
	aarch64? ( https://github.com/SChernykh/p2pool/releases/download/v${PV}/p2pool-v${PV}-linux-aarch64.tar.gz )
"
```

---

Oh that is surprising. I think I must have misread "x64" as "x86". I hate that confusing naming scheme.